test/integration/esys-tr-fromTpmPublic-session.int.c

```diff
@@ -1,4 +1,8 @@
 /* SPDX-License-Identifier: BSD-2-Clause */
+#ifndef HAVE_CONFIG_H
```

Comment:
This is a typo, it should be <code>#ifdef</code> instead of <code>#if*n*def</code>, otherwise the build fails.

> It is not different. It relays on MAXLOGLEVEL setting, it can come from config.h if this is autoconf based system which is guaranteed to be included in all files or from -DMAXLOGLEVEL= if in Windows. In any case it is guaranteed to have MAXLOGLEVEL available but not guaranteed to have config.h.

Agreed, I'm fine with the current approach of only including `config.h` in every source file.

Reply:
> This is a typo, it should be #ifdef instead of #ifndef, otherwise the build fails.

Sorry!!! Fixed.